### DISCUSSION TITLE
Alternative version

### DISCUSSION THREAD

**Discussion by MDylan** [schillermann/trolleydienst-php#25] [Show and tell]
Dear Brothers,

I have made an other software for public ministry service, based on Laravel.
It's support multilanguage too, currently Hungarian and English available. 
You can download it from here. 
https://github.com/MDylan/JW-public-ministry-organizer-app

Best regards,
David

**Comment by schillermann** [comment]
Hi @MDylan,

thank you for sharing your project.
I'll take a look at.

Warm greetings
Mario
